app/src/main/java/com/spascoding/taskycourse/feature_agenda/data/local/entities/relations/EventAndPhotos.kt

```diff
@@ -0,0 +1,15 @@
+package com.spascoding.taskycourse.feature_agenda.data.local.entities.relations
+
+import androidx.room.Embedded
+import androidx.room.Relation
+import com.spascoding.taskycourse.feature_agenda.data.local.entities.Event
+import com.spascoding.taskycourse.feature_agenda.data.local.entities.Photo
+
+data class EventAndPhotos(
+    @Embedded val event: Event,
+    @Relation(
+        parentColumn = "id",
+        entityColumn = "eventId"
+    )
+    val photos: List<Photo>
+)
```

Comment:
It's probably debatable, but I personally just implemented this photo list as a serialized JSON string part of the event table. We know that there are 10 photos at most per event and we also don't need any specific queries just for photos, so that approach has no downsides, but makes the whole DB setup a lot easier.

But your approach isn't wrong, so feel free to keep that